src/lib/OpenEXR/ImfMisc.cpp

```diff
@@ -275,6 +276,10 @@ copyIntoFrameBuffer (
     //
     endPtr += xStride;
 
+    // Make sure the copy loop below will terminate.
```

Comment:
Thanks! That's the check I am looking for :) Asserts only fire if the NDEBUG flag is set, and they'll terminate the program. Idiomatically we use them in OpenEXR to indicate programming errors. Since the inputs to this function could originate from user supplied inputs, in addition to coding errors, could you change the asserts to an exception please?

```cpp
            if (...)
            {
                throw IEX_NAMESPACE::ArgExc ("Stride will not exactly span input data.");
            }
```

Reply:
OK, I just realize these functions can be directly called externally.